internal/pkg/template/app.go

```diff
@@ -24,12 +29,22 @@ type AppNestedStackOpts struct {
 	PolicyOutputs   []string
 }
 
+// AppHealthCheckOpts holds configuration for the container healthcheck.
+type AppHealthCheckOpts struct {
+	Command     []string
+	Interval    int
+	Retries     int
+	StartPeriod int
+	Timeout     int
+}
+
 // AppOpts holds optional data that can be provided to enable features in an application stack template.
 type AppOpts struct {
-	// Environment variables.
 	Variables map[string]string
 	Secrets   map[string]string
 
+	HealthCheck *ecs.HealthCheck
```

Comment:
Do we ask the user for health check params, or point to the manifest and assume that they'll specify them there? 

We may be able to quickly add some healthcheck parsing from the dockerfile once this gets merged in so we can pass it directly to the manifest at creation.

Reply:
In the first iteration, we'll ship with them having to specify it in the manifest. And then like you said, we'll add the nice Dockerfile parsing to fill it up for them :)